website/docs/r/lb_service_monitor.html.markdown

```diff
@@ -0,0 +1,95 @@
+---
+layout: "vcd"
+page_title: "vCloudDirector: vcd_lb_service_monitor"
+sidebar_current: "docs-vcd-resource-lb-service-monitor"
+description: |-
+  Provides a NSX load balancer service monitor resource.
+---
+
+# vcd\_lb\_service\_monitor
+
+Provides a vCloud Director Edge Gateway Load Balancer Service Monitor resource. A service monitor 
+defines health check parameters for a particular type of network traffic. It can be associated with
+a pool. Pool members are monitored according to the service monitor parameters. 
+
+~> **Note:** To make load balancing work one must ensure that load balancing is enabled on edge gateway. This depends 
+on NSX version to work properly. Please refer to [VMware Product Interoperability Matrices](https://www.vmware.com/resources/compatibility/sim/interop_matrix.php#interop&29=&93=) 
+to check supported vCloud director and NSX for vSphere configurations.
+
+~> **Note:** The vCloud Director API for NSX supports a subset of the operations and objects defined in the NSX vSphere 
+API Guide. The API supports NSX 6.2, 6.3, and 6.4.
+
+Supported in provider *v2.4+*
+
+## Example Usage
+
+```hcl
+provider "vcd" {
+  user     = "${var.admin_user}"
+  password = "${var.admin_password}"
+  org      = "System"
+  url      = "https://AcmeVcd/api"
+}
+
+resource "vcd_lb_service_monitor" "monitor" {
+  org          = "my-org"
+  vdc          = "my-org-vdc"
+  edge_gateway = "my-edge-gw"
+
+  name        = "testing-monitor-yy"
+  interval    = "5"
+  timeout     = "20"
+  max_retries = "3"
+  type        = "http"
+  http_method = "GET"
+  url         = "health"
```

Comment:
more meaningful? e.g. http://192.168.10.10/health - Just suggesting :)

Reply:
The healthcheck should not imply hostname because load balancer server pool will specify list of servers and they will be checked. I can probably add `/health` for clarity.

Reply:
Yes, that would be nice :)

Reply:
Done

Reply:
It's still showin  `url         = "health"` for me

Reply:
seems valid :)